### PR TITLE
INFRA-88: added var for serv doc homepage url

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: gh-pages-stub/_config.yml [hunk 1]
```diff
@@ -7,11 +7,11 @@ permalink:        pretty
 #relative_permalinks: false
 
 # Setup
-title:            Kaa
+title:
 tagline:          'A Jekyll theme'
 description:      ''
 url:
-baseurl:          /kaa
+baseurl:
```
Comment (rasendubi) [inline gh-pages-stub/_config.yml]: Do not change baseurl and title. That'll break CI

File: gh-pages-stub/_includes/variables.md [hunk 2]
```diff
@@ -30,4 +30,5 @@
 {% assign component_mapping = site.data.component_mapping[version] %}
 {% assign sandbox_frame_url = component_mapping["Sandbox frame"] %}
 {% assign sample_apps_url   = component_mapping["Sample Apps"] %}
-{% assign rfc_url           = "https://github.com/kaaproject/kaa-rfcs/blob/master/" %}
\ No newline at end of file
+{% assign rfc_url           = "https://github.com/kaaproject/kaa-rfcs/blob/master/" %}
+{% assign doc_url           = "https://docs.kaaiot.io/" %}
```
Comment (rasendubi) [inline gh-pages-stub/_includes/variables.md]: Add trailing newline

Comment (rasendubi) [reply gh-pages-stub/_includes/variables.md]: Maybe, `docs_url`

**Comment by rasendubi** [comment]
Still not reverted.

**Comment by hunch7** [comment]
Fixed.

**Pull request merged**